client/quantum_serverless/utils/utils.py

```diff
@@ -0,0 +1,45 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+===========================================================
+Utilities (:mod:`quantum_serverless.utils.utils`)
+===========================================================
+
+.. currentmodule:: quantum_serverless.utils.utils
+
+Quantum serverless utilities
+====================================
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    utility functions
+"""
+
+import os
+import re
+
+
+def sanitize_file_path(path: str):
+    """sanitize file path.
```

Comment:
nit: let's document clearly what type of sanitizing we're doing, either here in the doc string or with comments in the code on each thing we clean